src/localization/locales/en-US.json

```diff
@@ -1016,6 +1016,18 @@
         "form_location_helper_text_step_3_image_src": "/storyMap/set-map-step-3-en.png",
         "form_location_helper_text_step_3_image_alt": "A screenshot of the Set Map Location screen. The “Set Location” button in the top right is circled.",
         "form_location_helper_text_bearing_icon_alt": "Bearing",
+        "form_location_add_data_layer_button": "Add Data Layer",
+        "form_location_add_data_layer_current": "Data Layer: <1>{{title}}</1>",
+        "form_location_add_data_layer_dialog_title": "<0>Add a data layer to </0><1>{{title}}</1>",
+        "form_location_add_data_layer_dialog_title_blank": "Add a data layer to chapter",
```

Comment:
question: is this working on the current change?

Reply:
@paulschreiber you mean `form_location_add_data_layer_dialog_title_blank`? Yes that is part of the title of the dialog